test/tests/styleTest.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("Zotero.Styles", function() {
+	var styleID = "http://www.zotero.org/styles/cell";
+	var styleFile = getTestDataDirectory();
+	var style;
+	styleFile.append('cell.csl');
```

Comment:
This is literally testing nsIFile handling in this function (as I was changing the signature and wanted to make sure old calls will remain functional)